packages/react/src/components/carbon-components-react/UIShell/SideNavMenu.js

```diff
@@ -177,7 +177,7 @@ export class SideNavMenu extends React.Component {
     if (item) {
       return React.cloneElement(item, {
         onClick:
-          item.props.isBackButton === true
+          item.props.isbackbutton === 'true'
```

Comment:
it's a boolean value but is checking against a string value?

Reply:
@jeffchew Yeah that was one of the React errors. It didn't like that a prop was placed on a DOM element and checking for a boolean. This is their suggestion to get around that.

Reply:
ok, so the proptype should be PropType.string instead of PropType.bool?

Otherwise looks good, I already approved it.

Reply:
@jeffchew Doh! Let me fix that...